application/messages/claim.go

```diff
@@ -97,13 +98,22 @@ func ClaimPreapprovedQueueMessage(tx *pop.Connection, claim models.Claim) {
 // ClaimReceiptQueueMessage queues messages to the claim's members to
 //  notify them that their claim requires receipts (again)
 func ClaimReceiptQueueMessage(tx *pop.Connection, claim models.Claim) {
-	claim.LoadPolicyMembers(models.DB, false)
-
-	// TODO Figure out how to tell the members what receipts are needed
+	claim.LoadPolicyMembers(tx, false)
+	claim.LoadClaimItems(tx, false)
+	clItem := claim.ClaimItems[0]
 
 	data := newEmailMessageData()
 	data.addClaimData(claim)
 
+	data["receiptMessage"] = ""
+
+	switch clItem.PayoutOption {
+	case api.PayoutOptionRepair:
+		data["receiptMessage"] = "Please provide the receipt(s) for repair costs."
```

Comment:
I personally do not like the "(s)" catch-all. In this case, I think the message could just be singular. (In other cases, it's worth the time to add an `if` to render singular or plural as needed.)
```suggestion
		data["receiptMessage"] = "Please provide a receipt for repair costs."
```